tcl/target/xtensa-core-esp32.cfg

```diff
@@ -0,0 +1,285 @@
+#  OpenOCD configuration file for Xtensa target architectures
```

Comment:
We can add core-esp32.cfg after merging esp32 to master.

Reply:
Removed